crates/nu-utils/Cargo.toml

```diff
@@ -17,9 +17,11 @@ bench = false
 bench = false
 
 [dependencies]
+fancy-regex = { workspace = true }
 lscolors = { workspace = true, default-features = false, features = ["nu-ansi-term"] }
 log = { workspace = true }
 num-format = { workspace = true }
+once_cell = { workspace = true }
```

Comment:
maybe i'm wrong but i thought this once_cell stuff was built into rust now?

https://github.com/rust-lang/rust/pull/105587
https://doc.rust-lang.org/nightly/std/cell/

Reply:
Oh, I didn't know that. I just moved the dependency from crate `nuon` to crate `nu-utils`, but I'll check if it can be removed entirely.

Reply:
I've taken a look, and we have multiple crates that depend on `once_cell` for `Lazy`. I'll make a separate PR to replace them with the now-built-in `LazyCell` once this gets merged.